### PR TITLE
SBERDOMA-975 fix ResidentTicket mutations and queries

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
Generate server schemas for custom mutations createResidentTicket, updateResidentTIcket and allResidentTicket.
Made a check for the existence of source id in the database, for this it was necessary to create server utilities TicketSource.
Wrote a test for the existence of the passed source id

### CODE REVIEW

File: apps/condo/domains/ticket/schema/CreateResidentTicketService.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ const CreateResidentTicketService = new GQLCustomSchema('CreateResidentTicketSer
         {
             access: true,
             type:
-                'type ResidentTicketOutput { organization: Organization!, property: Property!, unitName: String,' +
-                'sectionName: String, floorName: String, number: Int!, client: User!, clientName: String,' +
-                'clientEmail: String, clientPhone: String, details: String!, related: Ticket, isEmergency: Boolean, status: TicketStatus!' +
+                'type ResidentTicketOutput { organization: Organization, property: Property, unitName: String,' +
```
Comment (pahaz) [inline apps/condo/domains/ticket/schema/CreateResidentTicketService.js:22]: we always have organization it should be `!`. We always have property, number, details.

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/ticket/schema/CreateResidentTicketService.js (hunk 1 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/CreateResidentTicketService.js]: done in 44b66c2

File: apps/condo/domains/ticket/schema/CreateResidentTicketService.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  */
 const { Contact } = require('@condo/domains/contact/utils/serverSchema')
 const { Property } = require('@condo/domains/property/utils/serverSchema')
-const { Ticket } = require('../utils/serverSchema')
+const { Ticket, TicketSource } = require('../utils/serverSchema')
```
Comment (dkoviazin) [inline apps/condo/domains/ticket/schema/CreateResidentTicketService.js:6]: @domains

File: apps/condo/domains/ticket/schema/GetAllResidentTicketsService.js [hunk 3]
```diff
@@ -6,10 +6,12 @@ const { GQLCustomSchema } = require('@core/keystone/schema')
...
 const { generateQueryWhereInput } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+const { get } = require('lodash')
 
 const fieldsObj = {
     id: 'ID',
     number: 'Int',
+    property: 'Property',
     status: 'TicketStatus',
     source: 'TicketSource',
     classifier: 'TicketClassifier',
```
Comment (dkoviazin) [inline apps/condo/domains/ticket/schema/GetAllResidentTicketsService.js:17]: We have TicketPlaceClassifier and TicketCategoryClassifier 
classifier is deprecated

File: apps/condo/domains/ticket/schema/CreateResidentTicketService.js (hunk 2 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/CreateResidentTicketService.js:6]: Do we use domain import only when we import something from other domains and not from the same one?

File: apps/condo/domains/ticket/schema/GetAllResidentTicketsService.js (hunk 3 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/GetAllResidentTicketsService.js]: ok, updated input and output ResidentTicket with new classifiers types in 84598b9

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**